client/src/components/FoodSeeker/SearchResults/StakeholderPreview/StakeholderPreview.js

```diff
@@ -251,24 +292,210 @@ const StakeholderPreview = ({ stakeholder, handleFlyTo }) => {
 
               {!(stakeholder.inactiveTemporary || stakeholder.inactive) && (
                 <>
-                  {isOpenFlag && <Chip color="success" label="OPEN NOW" />}
-                  {isAlmostClosedFlag && (
+                  {isOpenFlag && (
                     <Chip
-                      color="mealProgram"
-                      label={`Closing in ${minutesToClosing} minutes`}
+                      color="success"
+                      label="Open Now"
+                      sx={{
+                        borderRadius: "6px",
+                        fontStyle: "normal",
+                        fontSize: "12px",
+                        margin: "16px 16px 0 0",
+                      }}
                     />
                   )}
+
                   {showAllowWalkinsFlag && (
-                    <Chip label="Walk-Ins Allowed" color="primary" />
+                    <Chip
+                      label="Walk-Ins Allowed"
+                      color="primary"
+                      sx={{
+                        backgroundColor: "#2260FF",
+                        borderRadius: "6px",
+                        fontStyle: "normal",
+                        fontSize: "12px",
+                        margin: "16px 0 0",
+                      }}
+                    />
                   )}
                 </>
               )}
             </Box>
+            {stakeholder.hours && stakeholder.hours.length > 0 ? (
+              <Accordion
+                disableGutters
+                sx={{
+                  ".MuiAccordionSummary-root": {
+                    padding: "0px",
+                    minHeight: "0px",
+                    marginTop: "16px",
+                    alignItems: "flex-start",
+                  },
+                  ".MuiAccordionSummary-root.Mui-expanded": {
+                    minHeight: "0px",
+                  },
+                  boxShadow: "none",
+                  "&.MuiAccordion-root:before": {
+                    display: "none",
+                  },
+                  ".MuiAccordionDetails-root": { padding: "0px 6px 0px 0px" },
+                }}
+              >
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1-content"
+                  id="panel1-header"
+                  sx={{
+                    ".MuiAccordionSummary-content": {
+                      justifyContent: "space-between",
+                      margin: "0px",
+                      flexWrap: "wrap",
+                    },
+                  }}
+                >
+                  <Typography>
+                    Open{" "}
+                    {nextOpenTime(
+                      stakeholder.hours.sort(hoursSort),
+                      tenantTimeZone
+                    )}
+                  </Typography>
+
+                  <Typography
+                    color="#4A80F5"
```

Comment:
In Figma, the `More Hours` link is black, do we plan to keep the link black or blue?

Reply:
It was black in an older design. If you look at the "Hover States update for left panel" section in Figma, it's blue